ui/src/app/store/pod/types.ts

```diff
@@ -83,7 +83,7 @@ export type BasePod = Model & {
   power_address: string;
   power_pass?: string;
   owners_count: number;
-  storage_pools: PodStoragePool[];
+  storage_pools?: PodStoragePool[];
```

Comment:
Might be good to leave a comment about why this might not be defined (it probably always should be?)

Reply:
Yep, I think this is really a bug in the websocket handler...